### DISCUSSION TITLE
Writing to sbyte KPlayerCurrentFieldID

### DISCUSSION THREAD

**Discussion by hexbyt3** [kwsch/PKHeX#4150] [Q&A] [answered]
in00b

**Comment by Lusamine** [comment, marked as answer]
This question is not really related to this repo.  PKHeX is a save editor, not a RAM editor. Different blocks in memory will be read/handled in different ways by the game, so you will need to research each one yourself. Often an edit doesn't "stick" because it's already been read into some other working memory. You're better off finding an automation community and asking there about RAM edits.

**Comment by kwsch** [comment]
None of the above code is PKHeX.Core, and PKHeX does not do any live edits. If you are using logic from a fork/plugin, ask there instead.
